services/thumbnails/README.md

```diff
@@ -39,6 +39,21 @@ If a file type was not properly assigned or the type identification failed, thum
 
 Thumbnails can either be generated as `png`, `jpg` or `gif` files. These types are hardcoded and no other types can be requested. A requestor, like another service or a client, can request one of the available types to be generated. If more than one type is required, each type must be requested individually.
 
+## Thumbnail Query String Parameters
+
+Clients can request thumbnail previews for files by adding `?preview=1` to the file URL. Requests for files with thumbnail availabe respond with HTTP status `404`.
+
+The following query parameters are supported:
+
+| Parameter | Required | Default Value                                        | Description                   |
+| --------- | -------- | ---------------------------------------------------- | ----------------------------- |
+| preview   | YES      | 1                                                    | generates preview             |
+| x         | YES      | first x-value configured in `THUMBNAILS_RESOLUTIONS` | horizontal target size        |
+| y         | YES      | first y-value configured in `THUMBNAILS_RESOLUTIONS` | vertical target size          |
```

Comment:
Same as for `x`, can be any numeric value. The thumbnail service will pick the closest of the defined resolution.